api-conversion/src/test/java/io/strimzi/kafka/api/conversion/converter/AbstractConversionTestBase.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.api.conversion.converter;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.strimzi.api.annotations.ApiVersion;
+
+import java.util.function.Function;
+
+abstract class AbstractConversionTestBase<V> {
```

Comment:
Why the json and cr conversion method split?
I guess it doesn't hurt ...

Reply:
It just seemed easier that way.It is not like I run into any issues with the original way or something. It makes it easier to handle the different options etc. as well when they are in separate commands.